src/Primitives/src/StringSegment.cs

```diff
@@ -261,17 +261,17 @@ public override int GetHashCode()
         /// <summary>
         /// Checks if two specified <see cref="StringSegment"/> have the same value.
         /// </summary>
-        /// <param name="left">The first <see cref="StringSegment"/> to compare, or <code>null</code>.</param>
-        /// <param name="right">The second <see cref="StringSegment"/> to compare, or <code>null</code>.</param>
-        /// <returns><code>true</code> if the value of <paramref name="left"/> is the same as the value of <paramref name="right"/>; otherwise, <code>false</code>.</returns>
+        /// <param name="left">The first <see cref="StringSegment"/> to compare, or <c>null</c>.</param>
+        /// <param name="right">The second <see cref="StringSegment"/> to compare, or <c>null</c>.</param>
+        /// <returns><c>true</c> if the value of <paramref name="left"/> is the same as the value of <paramref name="right"/>; otherwise, <c>false</c>.</returns>
         public static bool operator ==(StringSegment left, StringSegment right) => left.Equals(right);
 
         /// <summary>
         /// Checks if two specified <see cref="StringSegment"/> have different values.
         /// </summary>
-        /// <param name="left">The first <see cref="StringSegment"/> to compare, or <code>null</code>.</param>
-        /// <param name="right">The second <see cref="StringSegment"/> to compare, or <code>null</code>.</param>
-        /// <returns><code>true</code> if the value of <paramref name="left"/> is different from the value of <paramref name="right"/>; otherwise, <code>false</code>.</returns>
+        /// <param name="left">The first <see cref="StringSegment"/> to compare, or <c>null</c>.</param>
+        /// <param name="right">The second <see cref="StringSegment"/> to compare, or <c>null</c>.</param>
+        /// <returns><c>true</c> if the value of <paramref name="left"/> is different from the value of <paramref name="right"/>; otherwise, <c>false</c>.</returns>
```

Comment:
```suggestion
        /// <returns><see langword="true" /> if the value of <paramref name="left"/> is different from the value of <paramref name="right"/>; otherwise, <see langword="false" />.</returns>
```